index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
```

Comment:
"Consider adding a lang attribute to the html start tag to declare the language of this document." ~W3C Validator